ext/node/polyfills/net.ts

```diff
@@ -1440,7 +1440,6 @@ export class Socket extends Duplex {
    */
   override resume(): this {
     if (
-      this[kBuffer] &&
```

Comment:
Can you give an explanation why this is removed now?

Reply:
We don't use kBuffer for all handle reads (eg: TTY). This matches the condition with `override pause()` too.